src/components/manager.ts

```diff
@@ -356,8 +356,8 @@ export class Manager {
                 // We need to parse the fls to discover file dependencies when defined by TeX macro
                 // It happens a lot with subfiles, https://tex.stackexchange.com/questions/289450/path-of-figures-in-different-directories-with-subfile-latex
                 await this.parseFlsFile(this.rootFile)
-                this.extension.structureProvider.refresh()
                 this.extension.structureProvider.update()
+                this.extension.latexCommanderProvider.update()
```

Comment:
Define `LaTeXCommandTreeView.update` and call it instead of calling `LatexCommanderProvider.update` directly.